src/settings/Settings.js

```diff
@@ -160,6 +160,12 @@ export const SETTINGS = {
         displayName: _td("Show info about bridges in room settings"),
         default: false,
     },
+    "feature_invite_only_padlocks": {
+        isFeature: true,
```

Comment:
I wonder if instead of a feature flag (which should have docs) we could make this a boring setting, like the "Show hidden events" setting also on the labs page.

Reply:
Mmm, I've done it as a feature flag so it's easy to deploy enable / disable for develop if we decide to do that.

Reply:
Shall I add the usual docs then?

Reply:
Preferably, yea. I'm getting deja-vu about this conversation and locks in the right panel :s

Reply:
See https://github.com/vector-im/riot-web/pull/12173